### PR TITLE
adds minio support and status check

### PR DESCRIPTION
**Pull request by xiongnemo**
Minio support for Django: [link](https://github.com/py-pa/django-minio-storage)

* squash the commits

### CODE REVIEW

File: catfood/catfood/views.py [hunk 1]
```diff
@@ -43,3 +52,17 @@ def redis_connected(self):
...
             return False
+
+    def minio_connected(self):
+        try:
+            testfile_content = b'minio here'
+            path = default_storage.save('test/testfile', ContentFile(testfile_content))
+            print(path)
+            testfile_content_read = default_storage.open(path).read()
+            default_storage.delete(path)
+            if not (testfile_content == testfile_content_read):
```
Comment (DarkKowalski) [inline catfood/catfood/views.py:63]: Why?

File: catfood/catfood/views.py [hunk 2]
```diff
@@ -43,3 +52,17 @@ def redis_connected(self):
             return True
         except Exception:
             return False
+
+    def minio_connected(self):
+        try:
+            testfile_content = b'minio here'
+            path = default_storage.save('test/testfile', ContentFile(testfile_content))
+            print(path)
```
Comment (DarkKowalski) [inline catfood/catfood/views.py:60]: Debugging info should be removed here

File: catfood/catfood/views.py [hunk 3]
```diff
@@ -43,3 +52,17 @@ def redis_connected(self):
...
+            testfile_content_read = default_storage.open(path).read()
+            default_storage.delete(path)
+            if not (testfile_content == testfile_content_read):
+                return False
+            return True
+        except Exception as e:
+            print(e)
```
Comment (DarkKowalski) [inline catfood/catfood/views.py:67]: Use a logging function instead of print

File: .github/workflows/django.yml [hunk 4]
```diff
@@ -58,7 +73,7 @@ jobs:
       - name: Setup python environment # setting python environment to 3.x
         uses: actions/setup-python@v2
         with:
-          python-version: '3.x' # if you want multiple python version run just use matrix strategy in job config. See the documentation of GitHub Actions
+          python-version: "3.x" # if you want multiple python version run just use matrix strategy in job config. See the documentation of GitHub Actions
```
Comment (DarkKowalski) [inline .github/workflows/django.yml:76]: Why?

**Review by DarkKowalski — CHANGES_REQUESTED** [review]
Checkout my comments

File: catfood/catfood/views.py (hunk 1 above)
Comment (xiongnemo) [reply catfood/catfood/views.py]: Unneecessary statements removed

File: catfood/catfood/views.py (hunk 2 above)
Comment (xiongnemo) [reply catfood/catfood/views.py]: Debugging information removed

File: catfood/catfood/views.py (hunk 3 above)
Comment (xiongnemo) [reply catfood/catfood/views.py]: also removed as it's not necessary

File: .github/workflows/django.yml (hunk 4 above)
Comment (xiongnemo) [reply .github/workflows/django.yml]: I've beaten that stupid YAML extension

**Comment by xiongnemo** [comment]
Kindly check that if everything is on the track :)

**Pull request merged**